runner/sidecar/sources.go

```diff
@@ -83,6 +89,12 @@ func connectSources(ctx context.Context, process func(context.Context, []byte) e
 			defer span.Finish()
 			totalCounter.WithLabelValues(sourceName, fmt.Sprint(replica)).Inc()
 			totalBytesCounter.WithLabelValues(sourceName, fmt.Sprint(replica)).Add(float64(len(msg)))
+			t, ok := ctx.Value(dfv1.MetaTime).(int64)
```

Comment:
please disallow message without meta-data,  use `dfv1.GetMetaFromContext` to get this